src/index.html

```diff
@@ -15,8 +15,33 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+
+    <header class="header">
+     <a href="#imagem">
+      <img src="images/logo.png" alt="moyo" class="image">
+    </a>
+
+      <nav class="nav">
+        <ul>
+          <div class="iphone">
+          <li><a href="#item1" class="is-active">Iphone</a></li>
+        </div>
```

Comment:
Using a `<div>` element directly inside a `<ul>` is not semantically correct. According to HTML specifications, only `<li>`, `<script>`, and `<template>` elements are allowed as direct children of a `<ul>`. Consider using a `<li>` element instead of the `<div>` to maintain semantic correctness.